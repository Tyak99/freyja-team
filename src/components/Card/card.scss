$base-color: #ffffff;

.card {
  background: $base-color;
  border-radius: 5px;
  height: 400px;
  margin: auto;
  width: 400px;
  z-index: 1000;

  @media screen and (max-width: 420px) {
    width: 300px;
  }
}
