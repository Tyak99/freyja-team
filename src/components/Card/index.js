import card from './Card';

export default card;
