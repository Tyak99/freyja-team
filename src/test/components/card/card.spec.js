import React from 'react';
import { shallow } from 'enzyme';
import Card from '../../../components/Card/Card';

let Cards;

describe('Card component', () => {
  it('should render correctly the card component', () => {
    Cards = shallow(<Card />);
    expect(Cards).toMatchSnapshot();
  });
});
