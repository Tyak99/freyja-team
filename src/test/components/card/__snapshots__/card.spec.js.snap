// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render correctly the card component 1`] = `ShallowWrapper {}`;
